query-node/build.sh

```diff
@@ -34,3 +34,14 @@ yarn workspace query-node-mappings build
 
 cp ./generated/graphql-server/generated/schema.graphql ../storage-node/src/services/queryNode/schema.graphql
 cp ./generated/graphql-server/generated/schema.graphql ../distributor-node/src/services/networking/query-node/schema.graphql
+
+################################################
+# temporary patche TODO: create proper solution
+
```

Comment:
Ok, so it took a while to figure this out.

The `PROCESSOR_HOST` env var is available inside the `graphql-server`, how the problem is happening while dns [lookup](https://github.com/Joystream/hydra/blob/db6b2d4b611bf76de38160c1d800648f85e8f3de/packages/hydra-cli/src/templates/graphql-server/src/processorStateApp.ts.mst#L65), if the `graphql-server` is running with open telemetry instrumentation enabled, then the `lookup` function returns an address string (response type of callback-based function) instead of expected [LookupAddress](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/types/node/dns.d.ts#L72) object (response type of promise-based function). So, in short, open telemetry isn't applying a proper patch of the lookup function.

There are two possible solutions I can think of
- Don't  instrument the `dns` package in the `graphql-server` (a much simpler solution)
- Make a change in the `graphql-server` codebase to use `callback` based or `dns/promise` based `lookup` method instead of [promisify](https://github.com/Joystream/hydra/blob/master/packages/hydra-cli/src/templates/graphql-server/src/processorStateApp.ts.mst#L59) lookup function

Reply:
> * Don't  instrument the `dns` package in the `graphql-server` (a much simpler solution)

I think that is the best option to go with.  :+1:

Reply:
Addressed in [7c658a8](https://github.com/Joystream/joystream/pull/4779/commits/7c658a8000410da69fdc614458a33a951385d6e9)